pydantic/main.py

```diff
@@ -233,16 +235,27 @@ class BaseModel(metaclass=MetaModel):
         __fields__: Dict[str, Field] = {}
         __validators__: Dict[str, AnyCallable] = {}
         __config__: Type[BaseConfig] = BaseConfig
+        __root__: List[Any] = []
         _json_encoder: Callable[[Any], Any] = lambda x: x
         _schema_cache: 'DictAny' = {}
 
     Config = BaseConfig
     __slots__ = ('__values__', '__fields_set__')
 
-    def __init__(self, **data: Any) -> None:
+    def __init__(self, *__root__: Any, **data: Any) -> None:
```

Comment:
Humm, I've very unwilling to change the signature of `__init__`. 

Would it be possible to accept `__root__` directly to `parse_obj` only?

Reply:
I have reverted the  ``__init__`` method to master.
And I add a ``__root__`` argument in ``parse_obj``.
However, We can pass ``__root__`` object to ``__init__`` as a keyword argument.
```python
Model(__root__='a')
# or
Model.parse_obj(__root__='a')
```